source/miscellaneous/pumpkinCoconutCurryChickpeas.rst

```diff
@@ -0,0 +1,44 @@
+Pumpkin Coconut Curry with Chickpeas
+====================================
+
+Details
+-------
+
+* Number of servings: 6
+* Cook time: 30 minutes
+
+Ingredients
+-----------
+
+* Cooked rice
+* 1-3 tbsp oil
+* 1 onion, chopped
+* 1 bell pepper, chopped
+* 1 knob ginger, diced
+* 4 garlic cloves, minced
+* 1 1/2 tsp garam masala (or substitute curry powder)
+* 1 tsp ground cumin
+* 2 15 oz. cans chickpeas, rinsed
+* 1 13.5 oz can coconut milk (if using reduced fat coconut milk, don't add the water below)
+* 1 13.5 oz can pumpkin
+* 1/2 cup water
+* 1 tsp salt
+* 3/4 cup chopped cilantro
+* Fresh lime juice, from 1-2 limes
+* Naan
+
+
+Steps
+-----
```

Comment:
I am noticing the rest of the repo is inconsistent about this (@jennank, we should fix that), but the numbering should be `#.` instead of `1.` and so on. RST will automatically generate the numbers.